src/main/java/model/Lottos.java

```diff
@@ -0,0 +1,42 @@
+package model;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.counting;
+import static java.util.stream.Collectors.groupingBy;
+
+public class Lottos {
+
+    private final List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
```

Comment:
```suggestion
        this.lottos = new ArrayList(lottos);
```
위와 같이, [깊은 복사의 개념](https://rok93.tistory.com/entry/%EC%96%95%EC%9D%80%EB%B3%B5%EC%82%AC-VS-%EA%B9%8A%EC%9D%80%EB%B3%B5%EC%82%AC)을 이용해서, List의 값을 변경하여도 Lottos에 영향이 없도록 만들어 줘야합니다.

Reply:
생각지도 못한 부분이었던 것 같습니다. 좋은 지적 감사드립니다.